plugins/ci/pkg/ci/ci.go

```diff
@@ -77,6 +78,7 @@ func NewCIScan() (*CIScan, error) {
 	}
 
 	ci := CIScan{
+		autoScan:       cloneRepo,
```

Comment:
Are auto-scan, and whether the repository is being cloned, this reliably coupled? I am a bit confused by this line.

Reply:
Thanks. A comment in the struct for the `cloneRepo` member would help others understand - although non-blocking.

Reply:
yep, I've added the comment to the [cloneRepo variable](https://github.com/FairwindsOps/insights-plugins/pull/740/files#diff-044cde4902d1869165353b9fc0e73b91be6154df96d8e5ecdbd18fd7a3ed4326R61) as it is a local var